src/lib.rs

```diff
@@ -0,0 +1,11 @@
+//! Modeling the relations between objects.
+
+/// The error type used by the crate.
+pub type Error = failure::Error;
+
+/// The corresponding result type used by the crate.
+pub type Result<T> = std::result::Result<T, Error>;
```

Comment:
same question as `typed_index_collection`
Do we need to expose it ?
Does someone need to use this Result?

I surely missed something but it is already exposed in `src/relations.rs`

Reply:
Ah right, good point. Done in 55e339b89012a1bd1cd793a82d9a104b59504e1f.